### PR TITLE
Fix !clans, !raid commands

### LINKED ISSUE #197: Update clans to reflect Bards of Academia leadership change
**Is your feature request related to a problem? Please describe.**
Due to the Bards of Academia leadership change, !clans should not show Bards of Academia (thanks to RaniOfTheRain for pointing this out)
image

**Describe the solution you'd like**
Remove Bards of Academia from the response to !clans
The fix should be [this file](../blob/develop/commands/clans.js)

**Additional context**
This should be a quick fix - just noting it here so we don't forget about it with the pom war around the corner

### LINKED ISSUE #198: [BUG REPORT] !raid does not mention user properly
**Please Describe Your Issue**
Horace usually responds to !raid by mentioning the user who called it. However, Horace currently mentions their user ID instead. (thanks to Rani for reporting this)

**How to Reproduce**
Use !raid in #raid-room, observe that Horace responds with user ID
Screenshot of the issue 
image

**Expected behavior**
Horace should response with the username, not the user ID.

### PR DESCRIPTION
**Pull request by ThatNerdSquared**
`!raid` should now mention users when it's run. Still working on the `!faq` command, hopefully it'll be complete within the week. Closes #198 and #197.

### CODE REVIEW

File: commands/faq.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const faq = require('../data/faq.json');
+
+module.exports.execute = async (client, message) => {
+	//const channel = message.channel();
+	//message.channel.send(faq['raid-room']);
```
Comment (ghost) [inline commands/faq.js]: Can you throw these additions into a separate PR? Horace will send this in !help and don't want users confused about a non-existent command

File: commands/raid.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿module.exports.execute = async (client, message) => {
 	return await message.channel.send(
-		'**RAAAAAAAAAAAAAAAAAAAID⚔** with ' + message.author + '\nhttps://cuckoo.team/koa',
+		'**RAAAAAAAAAAAAAAAAAAAID⚔** with ' + `${message.author}` + '\nhttps://cuckoo.team/koa',
```
Comment (ghost) [inline commands/raid.js]: nitpick: can this be `<@${message.author.id}>` instead to prevent any future changes that may occur?

**Review by ghost — CHANGES_REQUESTED** [review]
does this fix focused raider as well?

File: commands/faq.js (hunk 1 above)
Comment (ThatNerdSquared) [reply commands/faq.js]: Should be good, couldn't figure this out until i realized i could just remove the file and send another commit 🤦‍♂️

**Review by ghost — APPROVED** [review]
Feel free to merge!

**Comment by ThatNerdSquared** [comment]
> does this fix focused raider as well?

I think that my other pull request (#195) should have fixed it, worked in my test server.

**Pull request merged**